src/stores/actor.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+/**
+ * This store helps to identify a the current actor in all cases.
+ * In Talk not every user is a local nextcloud user, so identifying
+ * solely by userId is not enough.
+ * If an as no userId, they are a guest and identified by actorType + sessionId.
+ */
+
+import { loadState } from '@nextcloud/initial-state'
+import { defineStore } from 'pinia'
+import { ATTENDEE, PARTICIPANT } from '../constants.ts'
+import { getTeams } from '../services/teamsService.ts'
+
+export const useActorStore = defineStore('actor', {
+	state: () => ({
+		userId: null,
+		sessionId: null,
+		attendeeId: null,
+		actorId: null,
+		actorType: null,
+		displayName: '',
+		actorGroups: loadState('spreed', 'user_group_ids', []),
+		actorTeams: [],
+	}),
+
+	getters: {
+		isActorUser: (state) => () => {
+			return state.actorType === ATTENDEE.ACTOR_TYPE.USERS
+		},
+
+		isActorGuest: (state) => () => {
+			return state.actorType === ATTENDEE.ACTOR_TYPE.GUESTS
+		},
```

Comment:
Creating a `computed` that computes a function doesn't make sense here.

There is no reactivity in the computation, or anything requiring lazy initiation.